Create ui-c8y-1023-14-84-filter-for-available-applications-on-tenant-details-applications-tab-shows-cockpit.md (#4264)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-14-84-filter-for-available-applications-on-tenant-details-applications-tab-shows-cockpit.md

---
date: ""

> title: Applications filter on tenant details now works with translated application names

Reviewer: ```suggestion
title: Application filter in tenant details now works with translated application names
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62149
version: 1023.14.84
---

> The applications filter on the tenant details Applications tab was previously matching against non-translated application names in the background, which meant that filtering by an application name in your currently selected language would not return any results. This has been fixed and the filter now correctly matches against the translated application names, allowing you to search for and find applications by their names as they appear in your selected language. This ensures a more intuitive filtering experience when managing applications for your tenants, as you can now reliably search using the application names visible in your user interface.

Reviewer: ```suggestion
The application filter on the **Applications** tab in the tenant details previously matched against non-translated application names in the background, which meant that filtering by an application name in your currently selected language would not return any results. This has been fixed, and the filter now correctly matches against the translated application names, allowing you to search for and find applications by their names as they appear in your selected language. This ensures a more intuitive filtering experience when managing applications for your tenants, as you can now reliably search using the application names visible in your user interface.
```

> The application filter on the **Applications** tab in the tenant details previously matched against non-translated application names in the background, which meant that filtering by an application name in your currently selected language would not return any results. This has been fixed, and the filter now correctly matches against the translated application names, allowing you to search for and find applications by their names as they appear in your selected language. This ensures a more intuitive filtering experience when managing applications for your tenants, as you can now reliably search using the application names visible in your user interface.

Reviewer: ```suggestion
The application filter on the **Applications** tab in the tenant details previously matched against non-translated application names in the background, which meant that filtering by an application name in your currently selected language would not return any results. This issue has been fixed, and the filter now correctly matches against the translated application names, allowing you to search for and find applications by their names as they appear in your selected language. This ensures a more intuitive filtering experience when managing applications for your tenants, as you can now reliably search using the application names visible in your user interface.
```
